draft-ma-opsawg-ucl-acl.md

```diff
@@ -394,6 +394,113 @@ file="ietf-schedule@2023-01-19.yang"
 <CODE ENDS>
 ~~~~
 
+
+### Examples
+
+   Here are some delicated examples of period and recurrence formats defined as
+   YANG groupings. The data is provided in JSON.
+
+#### Period of Time
+
+   The period starting at 08:00:00 UTC, on January 1, 2023 and ending at 18:00:00 UTC
+   on December 31, 2025:
+
+~~~~
+{
+  "period-of-time": {
+    "explicit-start": "2023-01-01T08:00:00Z",
+    "explicit-end": "2025-12-01T18:00:00Z"
+  }
+}
+~~~~
+
+   The period starting at 08:00:00 UTC, on January 1, 2023 and lasting 15 days and
+   5 hours and 20 minutes:
+
+~~~~
+{
+  "period-of-time": {
+    "start": "2023-01-01T08:00:00Z",
+    "duration": "P15DT05:20:00"
+  }
+}
+~~~~
+
+   The period starting at 08:00:00 UTC, on January 1, 2023 and lasting 20 weeks:
+
+~~~~
+{
+  "period-of-time": {
+    "start": "2023-01-01T08:00:00Z",
+    "duration": "P20W"
+  }
+}
+~~~~
+
+#### Recurrence Rule
+
+  Every day in December
+
+~~~
+{
+  "recurrence": {
+    "freq": "daily",
+    "bymonth": "12"
+  }
+}
+~~~  
+
+   10 occurrences that occur every last Saturday of the month:
+
+~~~~
+{
+  "recurrence": {
+    "freq": "monthly",
+    "count": "10",
+    "byday": {
+      "direction": "-1",
+      "weekday": "saturday"
+    }
+  }
+}
+~~~~     
+
+   The last workday of the month, until December 25, 2023:
```

Comment:
```suggestion
   The following indicates the example of a recurrence that occurs last workday of the month until December 25, 2023:
```

Reply:
Not sure to get this one. Also, isn't "bysetpos" defined as a leaf-list?

Reply:
Yup, leaf-list and integer - I think. It would limit the number of instances within a specific time period. The text might be improved with "on the"... But just checking the example to make sure it is the case.

"The following indicates the example of a recurrence that occurs on the last workday of the month until December 25, 2023:"

Reply:
Sorry, you guys are right, I will fix that.